src/main/java/org/agbrothers/englishbot/processing/processor/dictionary/PrintWordsDictionaryProcessor.java

```diff
@@ -0,0 +1,99 @@
+package org.agbrothers.englishbot.processing.processor.dictionary;
+
+import org.agbrothers.englishbot.entity.Word;
+import org.agbrothers.englishbot.processing.ProcessingExchange;
+import org.agbrothers.englishbot.processing.processor.Processor;
+import org.agbrothers.englishbot.service.DictionaryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.*;
+
+@Component
+public class PrintWordsDictionaryProcessor implements Processor {
+
+    private DictionaryService dictionaryService;
+
+    @Override
+    public void process(ProcessingExchange exchange)  {
+        String messageText = exchange.getMessageText();
+        List<Word> allWordsSorted = getAllWordsSorted();
+
+        if(allWordsSorted.size()<13){
+            exchange.setResponseMessageText(printWords(allWordsSorted));
+        }
+        else        {
+            exchange.setResponseMessageText(getResponseMessageText(allWordsSorted, messageText));
+            exchange.setResponseButtons(getKeyboardButtons(allWordsSorted));
+        }
+    }
+
+    private String getResponseMessageText (List<Word> list, String messageText) {
+        String currentWordEngVal;
+        if(getWordListsByFirstWord(list).containsKey(messageText)){
+            return printWords(getWordListsByFirstWord(list).get(messageText));
+        }
+        else {
+            return "У словнику більше 10 слів. Будь ласка виберіть діапазон слів для перегляду:";
```

Comment:
This is a bad practice. I must spend time to find out when the code runs into this `else` statement. You should explicitly specify the condition when to return this message. E.g.
```
if(ButtonLabel.PRINT_ALL_WORD.equals(messageText) {
   return "У словнику...
```

in this case, you can get rid of calling `getWordListsByFirstWord(list)` twice - which is really inefficient